.github/workflows/trymerge.yml

```diff
@@ -44,6 +44,8 @@ jobs:
           if [ -n "${REBASE}" ]; then
             python3 .github/scripts/tryrebase.py "${PR_NUM}" --branch "${REBASE}"
             git checkout master
+            # give github some time to get the push and start workflows
```

Comment:
nit: 
```suggestion
            # give github some time between the push and start workflows so that Github's messages
            # on the PR appear in chronological order (timing issues can otherwise shuffle them around)
```